src/test/java/seedu/address/model/widget/WidgetModelManagerTest.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.model.widget;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_HUSBAND;
+import static seedu.address.testutil.TypicalClients.AMY;
+import static seedu.address.testutil.TypicalClients.BOB;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.client.Client;
+import seedu.address.testutil.ClientBuilder;
+
+public class WidgetModelManagerTest {
+
+    @Test
+    void test_setContent_sameClient() {
```

Comment:
test functions should follow the format: featureUnderTest_testScenario_expectedBehavior()

e.g. sortList_emptyList_exceptionThrown() getMember_memberNotFount_nullReturned

Reply:
Changed and fixed checkystle!